src/pages/eunjungPark/Login/Login.js

```diff
@@ -1,8 +1,114 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './Login.scss';
 
 class LoginEunjung extends Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  handleInput = event => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  loginButton = () => {
+    const { id, pw } = this.state;
+    const checkId = id.includes('@');
+    const checkPw = pw.length >= 4;
+
+    if (checkId && checkPw) {
+      alert('입력성공');
+
+      fetch('http://10.58.3.176:8000/users/signin', {
+        method: 'POST',
+        body: JSON.stringify({
+          name: '이름',
+          email: this.state.id,
+          password: this.state.pw,
+          contact_mobile: '010-0000-0000',
+          nickname: 'AAA',
+          address: 'BBB',
+        }),
+      })
+        .then(response => response.json())
+        .then(response => {
+          if (response.message === 'SUCCESS') {
+            alert('로그인성공');
+            this.goToMain();
+          } else {
+            alert('로그인다시!');
+          }
+        });
+    } else {
+      if (!checkId) {
+        alert('아이디에는 @가 포함되어야 합니다.');
+      }
+
+      if (!checkPw) {
+        alert('비밀번호는 4자리 이상이어야 합니다.');
+      }
+    }
+  };
```

Comment:
- Early Return을 통해 코드의 상단에서 먼저 유효성 검사를 하시고, 검사를 통과하지 못하면 바로 return 문을 통해 함수의 실행을 종료하도록 코드를 구성해주시면 코드가 선형적으로 구성되어 가독성 측면에서 더 좋습니다 참고해주세요!
```suggestion
  loginButton = () => {
    const { id, pw } = this.state;
    const checkId = id.includes('@');
    const checkPw = pw.length >= 4;
    
    if (!checkId) return alert('아이디에는 @가 포함되어야 합니다.');
    if (!checkPw) return alert('비밀번호는 4자리 이상이어야 합니다.');

    alert('입력성공');

    fetch('http://10.58.3.176:8000/users/signin', {
      method: 'POST',
      body: JSON.stringify({
        name: '이름',
        email: this.state.id,
        password: this.state.pw,
        contact_mobile: '010-0000-0000',
        nickname: 'AAA',
        address: 'BBB',
      }),
    })
      .then(response => response.json())
      .then(response => {
        if (response.message === 'SUCCESS') {
          alert('로그인성공');
          this.goToMain();
        } else {
          alert('로그인다시!');
        }
      });
  };
```